pages/common/git-tag.md

```diff
@@ -0,0 +1,19 @@
+# git tag
+
+> Create, list, delete or verify tags
+
+- List all tags
```

Comment:
Well, yes, but since tags are usually connected to releases, I thought it could be a nice extra (but not a necessity) to clarify that upstream tags may not exist locally, so git tag won't list them even though they may exist. I guess my thinking was influenced by `git branch` vs. `git branch -r`, but maybe all remote tags are downloaded on cloning a repo, and the same isn't true for branches? I'm not sure if that's actually the case, btw.